### PR TITLE
enable user to enter q or Q to quit the program

### PR DESCRIPTION
**Pull request by xd3262nd**

### CODE REVIEW

**Comment by JHughes9802** [comment]
Works, but the if statement is unnecessary. Would recommend adding .strip() to choice.upper() in case the user accidentally taps the spacebar, especially because it's impossible to see if there's space(s) after the response (would help all responses, not just for quitting).

**Comment by xd3262nd** [comment]
@JHughes9802 I have tried without the if statement and it returned error. That's why I added the if statement. But I have added the `.strip()` to both of the choice input from the user

**Comment by JHughes9802** [comment]
That's interesting, the if statement on line 12 is necessary for you? I wonder if that has something to do with how Mac interprets data, because I'm able to run things just fine without it. Regardless, things run as intended.

**Comment by xd3262nd** [comment]
Yes if I want to do `choice.strip().upper()` to the `menu.isvalid()` method. It gave me a NoneType error.

**Comment by xd3262nd** [comment]
Okay I will just merge this branch then

File: ui.py [hunk 1]
```diff
@@ -10,9 +10,9 @@ def display_menu_get_choice(menu):
         choice = input('Enter choice? ')
 
         if type(choice) == str:
```
Comment (claraj) [inline ui.py:12]: There's no way for choice to not be a string. input return strings.

Comment (xd3262nd) [reply ui.py:12]: Yes, you are right. Totally miss that one out. Thanks for pointing it out!

**Pull request merged**